Library/Homebrew/os/mac/version.rb

```diff
@@ -76,7 +76,7 @@ def <=>(other)
       def to_sym
         @to_sym ||= begin
           # Big Sur is 11.x but Catalina is 10.15.
-          major_macos = if major >= 11
+          major_macos = if T.must(major) >= 11
```

Comment:
Sorbet question: what does this do and why is it needed?

Reply:
It casts `T.nilable(Type)` to `Type`, ensuring something is non-`nil`.

Reply:
What happens if it is `nil` with or without Sorbet enabled at runtime?

Reply:
Without, the `>=` call will fail; with, a `TypeError` is raised.

Both should never happen since the macOS major version should always be non-`nil`.

Reply:
Thanks. Is it possible to somehow move logic like this into where `major` is set/returned instead?

Reply:
It's calling `Version#major`, which definitely can have a `nil` major version.

Reply:
One option is to add an RBI file that says `OS::Mac::Version#major` will always return a non-nil `Token`

Reply:
> One option is to add an RBI file that says `OS::Mac::Version#major` will always return a non-nil `Token`

Something like that seems good 👍🏻